sirepo/mpi.py

```diff
@@ -69,6 +73,28 @@ def restrict_op_to_first_rank(op):
     return res
 
 
+def restrict_ops_to_first_node(ops):
```

Comment:
I think a different name would be helpful here. Specifically, "first_node". I don't know what that means. First in terms of what? Then the docstring says "current node". Are those the same thing? How does "first node", "current node", and "first rank" all relate to each other?

Reply:
I think we can take this discussion to a new issue.